custom_components/tahoma/climate_devices/atlantic_electrical_heater_with_adjustable_temperature_setpoint.py

```diff
@@ -177,8 +176,10 @@ def hvac_mode(self) -> str:
             return TAHOMA_TO_HVAC_MODE[
                 self.executor.select_state(CORE_OPERATING_MODE_STATE)
             ]
-        if CORE_ON_OFF_STATE in self.device.states:
-            return TAHOMA_TO_HVAC_MODE[self.executor.select_state(CORE_ON_OFF_STATE)]
+        if OverkizState.CORE_ON_OFF in self.device.states:
```

Comment:
I never noticed this, but in can happen this method will return... nothing. How can we nicely handle this case?

Reply:
log an error and return None?